apis/nucleus/src/object/lo-handler.js

```diff
@@ -54,13 +43,15 @@ export default function loHandler({ dc: lo, def, properties }) {
     removeMeasure() {},
 
     maxDimensions() {
-      return 1;
+      const { max } = def.dimensions || {};
+      const maxDims = typeof max === 'function' ? max() : max;
+      return maxDims || 1;
```

Comment:
I think this is wrong. addDimension do still not support muliple dimension.
I am questioning if it make sense to extend list object to support multiple dimension. I think it would be more natural to add a new type of data instead.


for filterpane (assuming we keep the current property structure) I think it would be more natural to add an type of data object. Not extending list object (or hyper cube)

Reply:
Yes, we can/should probably hard-code lo-handler to 1. Note though that the Filterpane uses the `lo-container-handler.js`, so we are not using `lo-handler.js` for multiple dims.